src/OpenTelemetry.Sampler.AWS/CHANGELOG.md

```diff
@@ -10,3 +10,6 @@ Initial release of `OpenTelemetry.Sampler.AWS`.
 
 * Update OpenTelemetry SDK version to `1.7.0`.
   ([#1486](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1486))
+
+* Make OpenTelemetry.ResourceDetectors.AWS and OpenTelemetry.Sampler.AWS AoT compatible.
```

Comment:
Might not want to mention ResourceDetectors.AWS in this changelog.

Reply:
```suggestion
* Make OpenTelemetry.Sampler.AWS native AoT compatible.
```